config/dev.exs

```diff
@@ -2,6 +2,7 @@ use Mix.Config
 
 config :oli,
   env: :dev,
+  problematic_query_detection: :disabled,
```

Comment:
We might want to set this up to read from `oli.env` environment configs, e.g. `System.get_env("DEV_PROBLEMATIC_QUERY_DETECTION_ENABLED")`, or simply always enable this in development so that someone doesn't accidentally check in a change here